src/app/pages/questions/services/audio-record.service.ts

```diff
@@ -72,4 +107,9 @@ export class AudioRecordService {
   failure(error) {
     this.logger.error('Error! ', error)
   }
+
+  cleanBase64String(base64Data: string): string {
+    // Remove all spaces and line breaks
+    return base64Data.replace(/\s+/g, '')
```

Comment:
why would there be whitespace in the data? Could it mean the data is not valid or corrupted?

Reply:
Ah yes I've removed this. I was using this to test something previously but we don't need this anymore.